extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -101,7 +101,7 @@ export class VariantAnalysisManager
     private readonly cliServer: CodeQLCliServer,
     private readonly storagePath: string,
     private readonly variantAnalysisResultsManager: VariantAnalysisResultsManager,
-    private readonly dbManager: DbManager,
+    private readonly dbManager?: DbManager, // the dbManager is only needed when the newQueryRunExperience is enabled
```

Comment:
I'd personally skip the comment here since we have the same in the caller. Up to you though!

Reply:
I added it so that when removing the flag all places appear that need to be touched when searching for it 🤷‍♀️